lib/model/document.rb

```diff
@@ -227,6 +227,28 @@ def handle_populations
           population['title'] = "Population #{index+1}"
         end
       end
+
+      # Remove exclusion and exception populations that are empty. The simple
+      # contains these empty populations, while the HQMF does not.
+      @populations.each do |population|
+        populations_to_remove = []
+        population.each do |key, value|
```

Comment:
you can do `population.reject!` here (https://ruby-doc.org/core-2.2.1/Hash.html#method-i-reject-21). That would allow you to get rid of `populations_to_remove` and the loop lower down.